K8s Release: Poseidon (#2585)
RedisLabs/redislabs-docs: content/kubernetes/active-active/global-config.md

categories: ["Platforms"]
aliases: {
    /kubernetes/active-active/preview/global-config/,
    /kubernetes/active-active/global-config/,

}
---

> The Redis Enterprise Active-Active database (REAADB) custom resource contains the field `.spec.globalConfigurations`. This field sets configurations for the Active-Active database across all participating clusters, such as memory size or shard count, as well as the global database secrets.

Reviewer: ```suggestion
The Redis Enterprise Active-Active database (REAADB) custom resource contains the field `.spec.globalConfigurations`. This field sets configurations for the Active-Active database across all participating clusters, such as memory size, shard count, and the global database secrets.
```
